superset/config.py

```diff
@@ -314,6 +314,10 @@ class CeleryConfig(object):
 # contain all the external tables
 CSV_TO_HIVE_UPLOAD_DIRECTORY = 'EXTERNAL_HIVE_TABLES/'
 
+# The namespace within hive where the tables created from
+# uploading CSVs will be stored
+UPLOADED_CSV_HIVE_NAMESPACE = 'superset'
```

Comment:
The new default changes the existing behvaior. You may want to considering setting this as None.

Reply:
Oops!